MANIFEST.in

```diff
@@ -0,0 +1 @@
+include requirements.txt
```

Comment:
I guess it's also a good practice to include the LICENSE file in the source distribution : 
```suggestion
include requirements.txt
include LICENSE
```